src/models/LogPrinter.ts

```
@@ -1,4 +1,11 @@
 import { safeStringify } from "./utils/safeStringify";
+
+// eslint-disable-next-line no-control-regex
+const ansiRegex = /[][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g;
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)